tests/SingleDatabaseTenancyTest.php

```diff
@@ -204,16 +204,16 @@
     Post::create(['text' => 'Foo', 'slug' => 'foo']);
     $data = ['text' => 'Foo 2', 'slug' => 'foo'];
 
-    $uniqueFails = Validator::make($data, [
+    $uniqueValidator = Validator::make($data, [
         'slug' => 'unique:posts',
-    ])->fails();
-    $existsFails = Validator::make($data, [
+    ]);
+    $existsValidator= Validator::make($data, [
```

Comment:
What's the change you made here? I don't see the point of moving the `->fails()` call to a later point and creating a validator instance.

Also, there's wrong code style on this line. And we generally don't use indented `->and()` in tests, it's better to just make two separate assertions, especially if there are `// comments` that would be vertically aligned (right below each other)

Reply:
> What's the change you made here? I don't see the point of moving the ->fails() call to a later point and creating a validator instance.

I see. Ok, I'll revert the changes to the same as it was. 

Your main question is, "What's the change you made here?". 

So these are the assertions. 
```php
// pest()->assertFalse($uniqueFails); // todo get these two assertions to pass. for some reason, the validator is passing for both 'unique' and 'exists'
// pest()->assertTrue($existsFails); // todo get these two assertions to pass. for some reason, the validator is passing for both 'unique' and 'exists'
```

I think these are basically wrong. We created a post with slug `foo`. And the new data is like this 
`$data = ['text' => 'Foo 2', 'slug' => 'foo'];`

So the **unique** rule should fail because the slug is not unique, and the **exists** rule should pass because the slug already exists. The final assertions look like this. 
```php
expect($uniqueFails)->toBeTrue(); // Expect unique rule failed to pass because slug 'foo' already exists
expect($existsPass)->toBeTrue(); // Expect exists rule pass because slug 'foo' exists
```

I've refactored the test.